Project.toml

```
@@ -0,0 +1,12 @@
+name = "TrixiSmartShockFinder"
+uuid = "6b42f25e-8cc0-41c2-936f-439385811e39"
+authors = ["Michael Schlottke-Lakemper <michael@sloede.com>"]
+version = "0.1.0"
+
+[deps]
+MuladdMacro = "46d2c3a1-f734-5fdb-9937-b9b9aeba4221"
+Trixi = "a7f1ee26-1774-49b1-8366-f1abc58fbfcb"
+
+[compat]
+MuladdMacro = "0.2.4"
+julia = "1.9"
```

```suggestion
julia = "1.8"
```
That's what we have in Trixi.jl

---

I am not sure I want that. IMHO we should just test this with the current Julia version and if it works, be done with it. I do not plan to support multiple Julia versions with this.

---

ok